set.mm

```diff
@@ -632959,6 +633115,92 @@ fixed reference functional determined by this vector (corresponding to
       ( cn wcel cgcd co gcdnncl mp2an ) AEFBEFABGHEFCDABIJ $.
   $}
 
+  ${
```

Comment:
Are there non-deduction versions of the theorems already available in the main body? Then they sould be referenced here in the comments, e.g. for ~ dvdstrd: "The divides relation is transitive, a deduction version of ~ dvdstrd ."